showcase/src/app/app.component.scss

```diff
@@ -0,0 +1,31 @@
+html,
+body {
+  height: 100%;
```

Comment:
I think this height is already set in `packages/stark-ui/assets/styles/_base.scss`... so I guess we can remove it from here right?